toolchain/gcc-9.3.0-06-with_libphoenix.patch

```diff
@@ -0,0 +1,105 @@
+diff -Naur gcc-9.3.0-orig/libstdc++-v3/configure gcc-9.3.0/libstdc++-v3/configure
+--- gcc-9.3.0-orig/libstdc++-v3/configure	2020-03-12 12:07:24.000000000 +0100
++++ gcc-9.3.0/libstdc++-v3/configure	2022-09-13 17:07:42.697560290 +0200
+@@ -899,6 +899,7 @@
+ with_target_subdir
+ with_cross_host
+ with_newlib
++with_libphoenix
+ enable_maintainer_mode
+ enable_shared
+ enable_static
+@@ -1653,6 +1654,7 @@
+                           configuring in a subdirectory
+   --with-cross-host=HOST  configuring with a cross compiler
+   --with-newlib           assume newlib as a system C library
++  --with-libphoenix       assume libphoenix as a system C library
+   --with-pic              try to use only PIC/non-PIC objects [default=use
+                           both]
+   --with-gnu-ld           assume the C compiler uses GNU ld [default=no]
+@@ -5447,6 +5449,13 @@
+ fi
+ 
+ 
++# Check whether --with-libphoenix was given.
++if test ${with_libphoenix+y}
++then :
++  withval=$with_libphoenix;
++fi
++
++
+   # Will set LN_S to either 'ln -s', 'ln', or 'cp -p' (if linking isn't
+   # available).  Uncomment the next line to force a particular method.
+   { $as_echo "$as_me:${as_lineno-$LINENO}: checking whether ln -s works" >&5
+@@ -29478,7 +29487,8 @@
+   # GLIBCXX_CHECK_MATH_SUPPORT
+ 
+   # First, test for "known" system libraries.  We may be using newlib even
+-  # on a hosted environment.
++  # on a hosted environment. There is also possibility to be using libphoenix,
++  # what is testing in the second test.
```

Comment:
I don't understand this clause.

Reply:
`configure` tests whether newlib or libphoenix is used as C library. If non of them is available, it will try to get to know about system library using the given target. I also simplified that comment.